docs/domains/Domain.md

```diff
@@ -20,6 +20,7 @@ DomainSpec is a description of a domain.
 | `clusters` | array of [Cluster](#cluster) | Configuration for the clusters. |
 | `configOverrides` | string | The name of the config map for optional WebLogic configuration overrides. |
 | `configOverrideSecrets` | array of string | A list of names of the secrets for optional WebLogic configuration overrides. |
+| `dataHome` | string | The in-pod name of the directory for data storage of default and custom file stores. |
```

Comment:
Maybe we can provide a more detail here?  For example, change "The" to "Optional", and add a sentence something like:  "If `dataHome` is not set or set to an empty string, data storage directories are determined from WebLogic domain home configuration.".

Reply:
Confirmed done/resolved.